source/_integrations/airtouch5.markdown

```diff
@@ -0,0 +1,26 @@
+---
+title: AirTouch 5
+description: Instructions on how to integrate the AirTouch 5 A/C controller into Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2024.2
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@danzel'
+ha_domain: airtouch5
+ha_platforms:
+  - climate
+ha_integration_type: integration
+---
+
+The AirTouch 5 integration allows you to control Ducted Air Conditioning Systems that are using the [AirTouch 5](https://www.airtouch.net.au/smart-air-conditioning/airtouch-5/) Controller.
```

Comment:
```suggestion
The AirTouch 5 integration allows you to control ducted air conditioning systems that are using the [AirTouch 5](https://www.airtouch.net.au/smart-air-conditioning/airtouch-5/) controller.
```
Tiny tweak. Applied sentence-style capitalization as this is not a proper noun, but a generic term. Based on Microsoft Style Guide on [capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization).